docs/templates/python/material/_base/attribute.html

```diff
@@ -0,0 +1,101 @@
+    {% set root_members = True %}
+  {% elif root_members %}
+    {% set show_full_path = config.show_root_members_full_path or config.show_object_full_path %}
+    {% set root_members = False %}
+  {% else %}
+    {% set show_full_path = config.show_object_full_path %}
+  {% endif %}
+
+  {% set attribute_name = attribute.path if show_full_path else attribute.name %}
+
+  {% if not root or config.show_root_heading %}
+    {% filter heading(
+        heading_level,
+        role="data" if attribute.parent.kind.value == "module" else "attr",
+        id=html_id,
+        class="doc doc-heading",
+        toc_label=('<code class="doc-symbol doc-symbol-toc doc-symbol-attribute"></code>&nbsp;'|safe if config.show_symbol_type_toc else '') + attribute.name,
+    ) %}
+
+      {% block heading scoped %}
+        {% if config.show_symbol_type_heading %}<code class="doc-symbol doc-symbol-heading doc-symbol-attribute"></code>{% endif %}
+        {% if config.separate_signature %}
+          <span class="doc doc-object-name doc-attribute-name">{{ attribute_name }}</span>
+        {% else %}
+          {%+ filter highlight(language="python", inline=True) %}
+            {{ attribute_name }}{% if attribute.annotation %}: {{ attribute.annotation }}{% endif %}
+            {% if attribute.value %} = {{ attribute.value }}{% endif %}
+          {% endfilter %}
+        {% endif %}
+      {% endblock heading %}
+
+      {% block labels scoped %}
+        {% with labels = attribute.labels %}
+          {% include "labels.html" with context %}
+        {% endwith %}
+      {% endblock labels %}
+
+    {% endfilter %}
+
+    {% block signature scoped %}
+      {% if config.separate_signature %}
+        {% filter format_attribute(attribute, config.line_length, crossrefs=config.signature_crossrefs) %}
+          {{ attribute.name }}
+        {% endfilter %}
+      {% endif %}
+    {% endblock signature %}
+
+  {% else %}
+
+    {% if config.show_root_toc_entry %}
+      {% filter heading(heading_level,
+          role="data" if attribute.parent.kind.value == "module" else "attr",
+          id=html_id,
+          toc_label=('<code class="doc-symbol doc-symbol-toc doc-symbol-attribute"></code>&nbsp;'|safe if config.show_symbol_type_toc else '') + attribute.name,
+          hidden=True,
+      ) %}
+      {% endfilter %}
+    {% endif %}
+    {% set heading_level = heading_level - 1 %}
+  {% endif %}
+
+  <div class="doc doc-contents {% if root %}first{% endif %}">
+    {% block contents scoped %}
+      {% block docstring scoped %}
+        {% with docstring_sections = attribute.docstring.parsed %}
+          {% include "docstring.html" with context %}
+        {% endwith %}
+      {% endblock docstring %}
+    {% endblock contents %}
+
+    {# START NEW CODE #}
+    {% if pipeline_steps(attribute_name) %}
+    {# https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks #}
+    <details class="success">
```

Comment:
I chose something other than info because that seemed to get a bit lost with the many other info blocks in our docs IIRC. Any others in here you like?

https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks

Reply:
Like these:
```
$ git grep "[!?+]\+ info" mne_bids_pipeline/_config.py | wc -l
      26
```
So I wanted to differentiate / standardize using something else. Looks like `note` isn't used so I'll try that?

Reply:
We can also add our own admonitions (what a word … I'll never remember it correctly!)

https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=ad#custom-admonitions

Reply:
Sure that would work, but we'd have to pick a color and icon. Maybe ✅ or 📋 icon? Actually `abstract` standard type (which we don't use anywhere) already uses the clipboard icon so that could be nice.

And maybe all the `Good practice / advice` that are currently `info` should be changed to `tip`?

Reply:
> Sure that would work, but we'd have to pick a color and icon. Maybe ✅ or 📋 icon? Actually `abstract` standard type (which we don't use anywhere) already uses the clipboard icon so that could be nice.

I would also suggest to use the current "Abstract" icon – not a custom unicode one.

But really it's something we can also adjust at any later time, it's not a blocker for me right now!